Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk:8-slim
+
+ARG FLEET_VERSION=compiler-fleet-1.0-SNAPSHOT
+
+ADD ./build/distributions/${FLEET_VERSION}.tar  .
+
+WORKDIR ./${FLEET_VERSION}/bin/
```

Comment:
I don't think that the leading dot makes any difference but in general you probably want absolute path.

Reply:
I tried to print smth in console and as I understood, it's already a absolute path. So it's located in root directory

Reply:
The path is relative, it works because WORKDIR before is / and thus this relative equals to absolute. I think it would not work if we had changed WORKDIR to some other value.

Reply:
If I run command `pwd` without WORKDIR it will print: `/`
If I leave WORKDIR as here, then I can obtain: `/papeeria-compiler-fleet-1.0-SNAPSHOT/bin`

As I know, pwd printing absolute path to directory

Reply:
Mikhail, what are you trying to prove? :) Yes, working dir by default is `/`. Yes, it works as it is now. This still won't make the path `./${FLEET_VERSION}/bin/` absolute. It is relative by definition because it starts with dot. If you insert instruction `WORKDIR /tmp` at line 2 (for whatever reason) it will put all stuff into `/tmp/papeeria-compiler-fleet-1.0-SNAPSHOT` instead of `/papeeria-compiler-fleet-1.0-SNAPSHOT`. Is it a bug or a feature? We (the readers) do not know, because we have no information in this file which could explain us, did you really mean to use relative paths in WORKDIR and ADD instructions, or you just didn't care and it worked by coincidence.

Reply:
Finally, I've checked the reference and understood what do you mean. 
I've changed paths to absolute.